app/scripts/migrations/121.1.test.ts

```diff
@@ -25,7 +32,8 @@ describe('migration #121.1', () => {
       },
     };
 
-    const newStorage = await migrate(oldStorage);
+    const newStorage = await migrate(cloneDeep(oldStorage));
```

Comment:
I updated the existing tests to use `cloneDeep` to ensure we don't accidentally forget to clone state in the migration itself. The clone happens in the migration in the copy+pasted portion, so it's not likely to be missed, but it's easy to cover in tests so why not.